oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -167,10 +167,18 @@ protected GoogleCredentials() {
    *
    * @param accessToken Initial or temporary access token.
    **/
-  public GoogleCredentials(AccessToken accessToken) {
+  protected GoogleCredentials(AccessToken accessToken) {
```

Comment:
For classes with a constructor that takes a single parameter, we may want to consider adding an `of(x)` function.

Reply:
Done